modules/rosa-how-upgrades-work.adoc

```diff
@@ -0,0 +1,56 @@
+
+[id="rosa-upgrade-scheduled-time_{context}"]
+== Cluster upgrade scheduled time
+
+You can schedule cluster upgrades by setting the scheduled time. This is when the preparation for the cluster upgrade begins with pre-upgrade health checks and additional compute capacity creation. The actual cluster upgrade starts within one hour from the scheduled time. You receive an email notification when the cluster upgrade starts.
+
+The Pre-Health Check (PHC) provides extra protection to ensure the scheduled update proceeds as expected and runs in the following two scenarios:
+
+* If an upgrade is scheduled for more than 2 hours from the current time, the PHC runs and the user is notified if there are any failures. This PHC is in the _New_ phase of the upgrade.
+* When the upgrade is immediate or within 2 hours, the PHC runs just before the upgrade begins. This PHC is in the _Upgrading_ phase of the upgrade.
+This means that PHC is always run at least one time during the upgrading phase but can also be run additionally in advance if the upgrade is scheduled for more than 2 hours from the current time.
+
+You can observe the status of the cluster upgrade by running the `rosa describe upgrade --cluster=<cluster name|cluster_id>` command in the ROSA CLI (`rosa`).
+
+[id="rosa-cluster-upgrade-overview_{context}"]
+== ROSA (classic architecture) upgrade overview
+
+The following are the high-level steps that occur during the ROSA (classic architecture) cluster update:
+
+. Scheduling the upgrade in advance triggers the `PreHealthCheck` and notifies users of any failures that they can then address before the upgrade begins.
+. Before the cluster upgrade begins, a cluster health check is performed by the MUO. If the MUO identifies an issue that requires corrective action, you will be notified. Some examples of the cluster health checks that MUO performs include:
+** Identifying any Pod Disruption Budgets (PDBs) that can potentially block or delay the update of the nodes.
+** Ensuring cluster Operators are available and healthy.
+** Ensuring cluster critical alerts are not firing.
+. A temporary compute node is created in the cluster to allow for the scheduling of drained pods during the update.
++
+[NOTE]
+====
+The temporary compute node creation does not always happen. For example, if there is no `worker` machine pool, the temporary compute node will not be created. This might happen when a cluster admin deletes the existing `worker` machine pool and creates another `worker` machine pool with a different name or instance type.
```

Comment:
@ravitri Is this correct? I thought temporary worker node is ALWAYS created. It is just that it is not created in workers machine pool because customer can delete it.

Reply:
What's documented is in fact correct. I stand corrected (because that was not my understanding). Now, I see this PR and it's documented correctly: https://github.com/openshift/managed-upgrade-operator/pull/382

Reply:
LGTM